update documentation "disk.enableUUID=1" (#866)
kubermatic/docs: content/kubeone/master/architecture/requirements/infrastructure_management/_index.en.md

[terraform-configs-github]: https://github.com/kubermatic/kubeone/tree/master/examples/terraform
{{% /notice %}}

> ## VMware

Reviewer: Let's put this above `Terraform Integration. Also, I would put a subheading that goes something like `Cloud Provider-specific Requirements`, and then `VMware vSphere` in that subheading.

```suggestion
## Cloud Provider-specific Requirements

### VMware vSphere
```

> For the vSphere csi plugin to work correctly machines created in vSphere need to meet the [prerequisites][vsphere-storage].

Reviewer: ```suggestion
For the vSphere CSI driver to work correctly, machines created in vSphere need to meet the [CSI driver prerequisites][vsphere-storage].
```

> This is done automatically by the machine-controller or the provided terraform examples, therefore it is only relevant when machines are created manually.

Reviewer: ```suggestion
This is done automatically by the machine-controller or the provided Terraform examples, therefore it is only relevant when machines are created manually.
```

> Make sure the that all VMs have the disk.enableUUID flag set to 1, this can be done using the [govc tool](https://github.com/vmware/govmomi/tree/master/govc) with the following command:

Reviewer: ```suggestion
Make sure that all VMs have the `disk.enableUUID` flag set to 1. This can be done using the [govc tool](https://github.com/vmware/govmomi/tree/master/govc) with the following command:
```

Also, please make sure the URL is handled in the same way as other URLs.

```bash
govc vm.change -e="disk.enableUUID=1" -vm='/PATH/TO/VM'
```

> To support all features of the csi plugin ensure that the virtual machine hardware version is 15 or later and VMware Tools are installed.

Reviewer: This is required, so I'd recommend phrasing it like that, for example:

```suggestion
The vSphere CSI driver requires that the Virtual Machine Hardware Version is 15 or newer, and the VMware Tools are installed.
```
